renovate.json

```diff
@@ -7,6 +7,11 @@
       "matchUpdateTypes": ["minor", "patch"],
       "autoApprove": true,
       "automerge": true
+    },
+    {
+      "matchPackageNames": ["lz4"],
+      "prBodyNotes": ":warning: Check aarch64 compatibility :warning:",
+      "autoApprove": false
```

Comment:
This can happen any time with any other package. What if we would just auto approve minor versions and major versions should be manually accepted?

Reply:
this was a minor version update